src/sentry/utils/query.py

```diff
@@ -237,3 +237,12 @@ def bulk_delete_objects(
         )
 
     return has_more
+
+
+def drop_column_indexes(schema_editor, model, indexes):
```

Comment:
Maybe `column_names` instead of indexes? Or `indexed_columns`?

Reply:
Also worth having a comment that explains what this does. Something along the lines of:
"Accepts a list of columns and removes any indexes that are only relevant for those columns.". Wording is a bit hard...